CONTRIBUTING.md

```diff
@@ -19,6 +19,17 @@ frustration later on.
 All submissions, including submissions by project members, require review. We
 use Github pull requests for this purpose.
 
+Please see [the xi-editor contribution guidelines](https://github.com/google/xi-editor/blob/master/CONTRIBUTING.md) about guidelines for contributors with write access.
+
+If your name does not already appear in the [AUTHORS] file, please feel free to
```

Comment:
Is this supposed to link to an included authors file, or to the author's file for xi-editor?